### PR TITLE
[Relay] Add constant support.

### PR DESCRIPTION
**Pull request by cgyurgyik**
Supports constants defined in `let` bindings.
Also includes some small additional small clean-up.

### CODE REVIEW

File: frontends/relay/dahlia_impl.py [hunk 1]
```diff
@@ -441,6 +441,9 @@ def emit_components(func_defs: List[DahliaFuncDef]) -> str:
     created from the list of Dahlia function definitions.
     This does not include the import statement.
     """
+    if len(func_defs) == 0:
```
Comment (sampsyo) [inline frontends/relay/dahlia_impl.py:444]: I think `if not func_defs` is equivalent.

**Review by sampsyo — COMMENTED** [review]
Looks very cool! One silly Python comment within. :smiley:

File: frontends/relay/dahlia_impl.py (hunk 1 above)
Comment (cgyurgyik) [reply frontends/relay/dahlia_impl.py]: Fixed. Thanks!

**Comment by rachitnigam** [comment]
LGTM!

**Pull request merged**